sockeye.tconf

```diff
@@ -76,7 +43,7 @@ global {
   resource_flags_decode="-q gpu.q -l gpu=1,mem_free=4g"
 
   # SGE: flags for notifying about job completion (put in your email address!)
-  action_flags="-m ae -M YOUR_EMAIL_HERE"
+  action_flags="-m ae -M post@cs.jhu.edu"
```

Comment:
You probably don't want to push anything with your email address. People are generally not careful enough to change it to their own.

I learned it the hard way by getting 1000+ emails in my mailbox :p